### PR TITLE
Add pretty print for dump

### PR DESCRIPTION
**Pull request by jdegoes**

### CODE REVIEW

**Comment by jdegoes** [comment]
@iravid Agreed! We should integrate that into zmx, a machine readable mode for all the dumps and metrics.

**Comment by softinio** [comment]
> @iravid Agreed! We should integrate that into zmx, a machine readable mode for all the dumps and metrics.

I'll work on that next week

File: core/shared/src/main/scala/zio/Fiber.scala [hunk 1]
```diff
@@ -417,7 +418,55 @@ object Fiber {
...
+        case Running => "Running"
+        case Suspended(interruptible, epoch, _, asyncTrace) =>
+          val in = if (interruptible) "interruptible" else "uninterruptible"
+          val ep = s"${epoch} asyncs"
+          val as = asyncTrace.map(_.prettyPrint).mkString(" ")
+          s"Suspended(${in}, ${ep}, ${as})"
+      }
+
+      s"""
+      ${name}#${fiberId.seqNumber} (${lifeMsg}) ${waitMsg}}
```
Comment (regiskuckaertz) [inline core/shared/src/main/scala/zio/Fiber.scala]: Don't you have to use `stripMargin` here?

Comment (jdegoes) [reply core/shared/src/main/scala/zio/Fiber.scala]: Thank you! 🙏

Comment (regiskuckaertz) [reply core/shared/src/main/scala/zio/Fiber.scala]: No worries 🤗  You will have to prefix with `|` every line though, like this:
```scala
s"""
   |${name}#${fiberId.seqNumber} (${lifeMsg}) ${waitMsg}}
   |  Status: ${statMsg}
   |    ${trace.prettyPrint}
   |""".stripMargin

**Pull request merged**